### PR TITLE
[Doc Feature][Mute and unmute apps in meeting stage][2507281]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e3a98723eb2e5819516c728405e744809c08868b/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 87 | 39 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a5262e1-90da-43a2-862d-919e9c4a47d3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8652b19cace627a88f77ab2c50c850df502bdad9/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 88 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1ad43587-aefc-4289-b0e9-93c1bfdeb521) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e895e3dfd81712da3ac1152b97822879d88cd20c/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 88 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c6550afc-05fe-48f8-943f-3273a83b6316) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a4e93f561447109490cb41ae0c8b0b7e18efc799/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 88 | 40 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d908d33a-5f80-4fd8-8f03-9e7d5d344d58) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/01cb1ecddf115293e85cc50f47af1c4f378fdafb/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 90 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cef60f99-536e-493a-9f52-7415b8a93927) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b6a84db302a5dd780797b2d88386a2ac0d8b7fbd/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 90 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bbe6f098-ef8a-4943-abf6-d191f8e751ca) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e681095ccd3cad2559f25ac7b7bcf6d9d0bc3e18/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 90 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8acf896c-526d-43fa-af69-2eadd3f3b397) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5f7c227f47f167e39e27cbaea1285cbd5bf8b5f1/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 90 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/884a10ae-d0d9-4c24-a7f0-b301a7d4e6cb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c86491222585e9570f2c8d7d3f54c767ca04d43b/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 90 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d5e1d7f6-04bd-43f2-99dc-ac2f909bc2eb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c86491222585e9570f2c8d7d3f54c767ca04d43b/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7cf1ce91-e949-45bb-a125-e009e6197c7d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/96e5ec53c626b0c2ddb5136248fcee27cc407a4f/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f05b24ad-6e06-4ff1-be2a-be1a137b0864) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/96e5ec53c626b0c2ddb5136248fcee27cc407a4f/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5a89f1ef-bee1-4a3a-bec6-ffad8395c497) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ac1daa5367190a7b4ac388596ffea5b9a3f83ad2/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/91a30195-91d4-48b6-a439-2d4f0a347620) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ac1daa5367190a7b4ac388596ffea5b9a3f83ad2/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/51aec5e0-14dd-4d7d-a81b-d8a4ba97e500) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/26f05339ef808e66963245c745a7932c92b269d7/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1e48234c-8831-4adf-a18e-d7de5975962d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/26f05339ef808e66963245c745a7932c92b269d7/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/56b50463-c3fc-4be1-b532-a8e15e4e5a4d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/de104092d0d27bb7d10f374de60f428ffbb72763/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 93 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cfb92b43-9f8b-45d4-88b7-6f4180d4c1ee) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/de104092d0d27bb7d10f374de60f428ffbb72763/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/20504b3f-8c36-49b7-ba9b-3fe76303947c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a7c6316f9b5fa4d10ebcde279c6e4957772a8d2/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 93 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a3219308-cbf3-4fa3-be38-95aa90325b4b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a7c6316f9b5fa4d10ebcde279c6e4957772a8d2/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0efa197f-1566-4307-82c4-ae7f60567430) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a6482a5e372897088501064df283de39e716ac3/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 93 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/91748549-e919-4aeb-b625-57b943860b76) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a6482a5e372897088501064df283de39e716ac3/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7428004e-f3ef-4b2d-be79-e22695d19c63) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/76a729dd327c181933189dc926c97e96c073fd9d/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 93 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4146349d-ce87-403a-9661-16af97bac483) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/76a729dd327c181933189dc926c97e96c073fd9d/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c56f9b10-a6a0-4c39-bdad-7399f706d04d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4e5c94897748d51351c4dc72f4b46d99ce2405fd/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 93 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4d13bed1-3b5c-40cf-8c8c-e7c42deeffbc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4e5c94897748d51351c4dc72f4b46d99ce2405fd/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e16a2a96-1b56-4a7f-921f-d9d8e1e2bfee) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by SupritiBhan — APPROVED** [review]

**Review by v-habibaumme — APPROVED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9daf0bd0fe98462d8f04dc66b28c86afa7d6c690/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/19910938-7ff6-4f3a-b1f8-71f6e5ea5d7c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9daf0bd0fe98462d8f04dc66b28c86afa7d6c690/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a450c43d-b661-4234-ab5c-e546d44620ec) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/da74b9a71b07b3cd1395211aacc84c6d5b388864/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a74e7731-027d-47a4-b00c-17c110396a75) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/da74b9a71b07b3cd1395211aacc84c6d5b388864/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e8f487f3-9c53-4bf9-a57d-1c60caff07a0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by SupritiBhan — DISMISSED** [review]

File: msteams-platform/apps-in-teams-meetings/API-references.md [hunk 1]
```diff
@@ -925,6 +927,84 @@ The following code provides an example of meeting end event payload:
...
+## Toggle incoming audio
+
+The `toggleIncomingClientAudio` API allows an app to toggle the incoming audio speaker setting for the meeting user from mute to unmute or vice-versa. The API is available through the Teams client SDK.
+
+> [!NOTE]
+> The `toggleIncomingClientAudio` API for mobile is currently available only in [Public Developer Preview](../resources/dev-preview/developer-preview-intro.md).
+
+### Query parameter
+
+The following table includes the query parameters:
```
Comment (ashishguptaiitb) [inline msteams-platform/apps-in-teams-meetings/API-references.md:979]: Only 1 parameter so singular.

File: msteams-platform/apps-in-teams-meetings/API-references.md [hunk 2]
```diff
@@ -496,7 +498,7 @@ The JSON response body for Meeting Details API is as follows:
 | **details.scheduledEndTime** | The meeting's scheduled end time, in UTC. |
 | **details.joinUrl** | The URL used to join the meeting. |
 | **details.title** | The title of the meeting. |
-| **details.type** | The meeting's type - e.g. GroupCall, OneToOneCall, Adhoc, Broadcast, MeetNow, Recurring, Scheduled, Unknown. |
+| **details.type** | The meeting's type. For example, GroupCall, OneToOneCall, Adhoc, Broadcast, MeetNow, Recurring, Scheduled, or Unknown. |
```
Comment (ashishguptaiitb) [inline msteams-platform/apps-in-teams-meetings/API-references.md:501]: Is this a complete list of types of meetings? If yes, then don't add `For example`.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b9feb02dfd9aa0075f402c43a3a6f06a3a039b2c/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c6d72d59-357e-40f6-9a9c-b0fe32c788b4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b9feb02dfd9aa0075f402c43a3a6f06a3a039b2c/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1b4b619f-82f1-485d-ba48-315d7a850b1e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ashishguptaiitb — APPROVED** [review]

**Pull request merged**